app/controllers/content_items_controller.rb

```diff
@@ -97,6 +97,20 @@ def load_taxonomy_navigation
                                .yield_self { |document_collections| document_collections || [] }
                                .select { |document_collection| document_collection['document_type'] == 'document_collection' }
                                .map { |document_collection| document_collection.values_at('base_path', 'title') }
+
+      @banner_items = @content_item
```

Comment:
Could we extract this to give it some context?  It's a bit hard to mentally parse something this long, so might benefit from pulled out and given a name?